Add explanation text for yaml file naming (#383)
owncloud/docs-ocis: modules/ROOT/partials/deployment/services/env-and-yaml.adoc

=== YAML Example

> Note that the filename shown below is by purpose. +

Reviewer: ```suggestion
Note that the filename shown below has been chosen on purpose. +
```
